.github/workflows/ci.yml

```
@@ -0,0 +1,47 @@
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.13'
+
+      - name: Install Pipenv
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)